python/ray/memory_monitor.py

```diff
@@ -114,6 +114,11 @@ def raise_if_low_memory(self):
                 with open("/sys/fs/cgroup/memory/memory.usage_in_bytes",
                           "rb") as f:
                     used_gb = int(f.read()) / (1024**3)
+                # Exclude the page cache
+                with open("/sys/fs/cgroup/memory/memory.stat", "r") as f:
+                    for line in f.readlines():
+                        if line.split(" ")[0] == "cache":
+                            used_gb = used_gb - int(line.split(" ")[1]) / (1024**3)
```

Comment:
Can we add an assert this doesn't cause used_gb to go negative?

Reply:
Done! I already push my change to my branch, will it not sync to this pull  @ericl

https://github.com/yncxcw/ray/commit/1d6283609d4e45b27dc990b21b1fe4eda3511554

Reply:
Done! I already push my change to my branch, will it not sync to this pull  @ericl

https://github.com/yncxcw/ray/commit/1d6283609d4e45b27dc990b21b1fe4eda3511554

Reply:
@ericl Can you take a look again ?